tests/sentry/eventstream/test_eventstream.py

```diff
@@ -336,6 +339,47 @@ def test_transaction_queue(self, mock_eventstream_insert):
         assert "occurrence_id" not in dict(headers)
         assert body["queue"] == "post_process_transactions"
 
+    @override_settings()
+    @patch("sentry.eventstream.backend.insert", autospec=True)
+    def test_queue_split_router(self, mock_eventstream_insert):
```

Comment:
Do we want a test for default queue assignment?

Reply:
Added one for errors.